src/main/java/com/taskadapter/redmineapi/bean/Issue.java

```diff
@@ -43,6 +43,7 @@
     private String statusName;
     private Version targetVersion;
     private IssueCategory category;
+    private Boolean isPrivate;
```

Comment:
how about calling it "privateIssue" instead? having "is" prefix goes against Java Beans conventions